pusher-platform-android/src/main/kotlin/com/pusher/platform/schedulers.kt

```diff
@@ -1,87 +0,0 @@
-package com.pusher.platform
-
-import android.os.Handler
-import android.os.Looper
-import java.util.concurrent.LinkedBlockingQueue
-import java.util.concurrent.ThreadPoolExecutor
-import java.util.concurrent.TimeUnit
-
-typealias Action = () -> Unit
-
-class BackgroundScheduler(
```

Comment:
Using `Futures` and their extensions with a private `ExecutorService` we provide the same functionality without trying to reinvent the wheel.